src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -1,4 +1,75 @@
 package com.kodilla.ecommercee.domain;
 
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "PRODUCTS")
 public class Product {
+    private long id;
+    private String name;
+    private String description;
+    private BigDecimal price;
+    private Group group;
```

Comment:
Jeżeli w Group stosujemy HashSet, to w Produkt nie powinniśmy określić wedle jakiego kryterium produkt będzie porównywany, tzn nadpisać hashCode i equals?

Reply:
Tak. Masz rację.

Reply:
Poprawiłem